plugins/arDominionB5Plugin/templates/_footer.php

```diff
@@ -0,0 +1,32 @@
+<footer>
+
+  <?php if (QubitAcl::check('userInterface', 'translate')) { ?>
+    <?php echo get_component('sfTranslatePlugin', 'translate'); ?>
+  <?php } ?>
+
+  <?php echo get_component_slot('footer'); ?>
+
+  <div id="print-date">
+    <?php echo __('Printed: %d%', ['%d%' => date('Y-m-d')]); ?>
+  </div>
+
+  <div id="js-i18n">
+    <div id="read-more-less-links"
+      data-read-more-text="<?php echo __('Read more'); ?>" 
+      data-read-less-text="<?php echo __('Read less'); ?>">
+    </div>
+  </div>
+
+</footer>
+
+<?php $gaKey = sfConfig::get('app_google_analytics_api_key', ''); ?>
+<?php if (!empty($gaKey)) { ?>
+  <script <?php echo __(sfConfig::get('csp_nonce', '')); ?> async src="https://www.googletagmanager.com/gtag/js?id=<?php echo $gaKey; ?>"></script>
+  <script <?php echo __(sfConfig::get('csp_nonce', '')); ?>>
```

Comment:
Do we need to duplicate the footer in B5 just to add the nonce here? Would it work adding it directly to the existing footer?

Reply:
Getting the CSP nonce should not happen in BS2 as it is unsupported. It would have no effect to retrieve the nonce as it would evaluate to '', but I felt it was confusing to have that code in templates that could be run under BS2. I am open to changing this.

Reply:
Not sure if it happened in the other PR, but we are doing it in this one in the tag manager template. I'd prefer not duplicating the templates if possible, but not a big deal.

Reply:
I see - will revert this bit and add the nonce in the other footer file